firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -149,28 +150,64 @@ public void incrementTsnsCount(int value) {
     tsnsCount.addAndGet(value);
   }
 
-  @Override
-  public void onActivityCreated(Activity activity, Bundle savedInstanceState) {
+  // Starts tracking the frame metrics for an activity.
+  public void startFrameMonitoring(Activity activity) {
```

Comment:
Does this need to be public?

Reply:
Doesn't have to be. Marking this as `private`.